superset-frontend/tsconfig.json

```diff
@@ -27,7 +27,7 @@
       ],
       // for supressing errors caused by incompatible @types/react when `npm link`
       // Ref: https://github.com/Microsoft/typescript/issues/6496#issuecomment-384786222
-      "*": ["./node_modules/@types/*", "*"]
+      "react": ["./node_modules/@types/react"]
```

Comment:
needed because we now have two different version of "match-sorter", one in root, another from the table viz.